modules/raft/src/main/java/org/apache/ignite/internal/raft/server/impl/JraftServerImpl.java

```diff
@@ -121,8 +122,18 @@ public JraftServerImpl(ClusterService service, Path dataPath, NodeOptions opts)
         this.opts.setSharedPools(true);
 
         if (opts.getServerName() == null) {
-            opts.setServerName(service.localConfiguration().getName());
-        }
+            this.opts.setServerName(service.localConfiguration().getName());
+        }
+
+        // Timeout increasing strategy for election timeout. Adjusting happens according to
+        // {@link org.apache.ignite.raft.jraft.util.ExponentialBackoffTimeoutStrategy} when a leader is not elected, after several
+        // consecutive unsuccessful leader elections, which could be controlled through
+        // {@link org.apache.ignite.raft.jraft.option.NodeOptions.MAX_ELECTION_ROUNDS_WITHOUT_ADJUSTING}
+        // Max timeout value that {@link org.apache.ignite.raft.jraft.util.ExponentialBackoffTimeoutStrategy} could produce is
+        // 11s. This value must be more than timeout of a membership protocol to remove failed node from the cluster.
+        // In our case, we may assume that 11s could be enough as far as 11s is greater
```

Comment:
Why do we have constants like 11s in javadoc? This is a configurable propery.

Reply:
I think there is important to mention that this 11s is not a random value, but produced from suspicion timeout